src/move/visualization/style.py

```diff
@@ -28,11 +28,11 @@ def color_cycle(colormap: str) -> ContextManager:
     Returns:
         Context manager
     """
-    registry: ColormapRegistry = getattr(matplotlib, "colormaps")
+    registry: ColormapRegistry = matplotlib.colormaps
     colormap = registry[colormap]
     if not isinstance(colormap, ListedColormap):
         raise ValueError("Only colormaps that are list of colors supported.")
-    prop_cycle = cycler(color=getattr(colormap, "colors"))
+    prop_cycle = cycler(color=colormap.colors)
```

Comment:
The reason for doing `getattr` instead of just indexing is to prevent the pylance from complaining. Because matplotlib colormaps don't have a `colors` attribute by default (it is added during run at some point). So Pylance will complain: "Cannot acces attribute `colors` for class `Colormap`. Attribute `colors` is unkwown"

Reply:
So I should revert that change? Otherwise we can just add a comment to ignore that bug. It does not pop up in my editor- which matplotlib version are you currently using?

Reply:
Perhaps they have better type hints nowadays, the package is very messy in that regard. I'm using matplotlib 3.8.0

Reply:
I am on `3.9.0`

Reply:
I have updated to 3.9.0, but I am getting the same error message. 🤨

Reply:
Fixed